shared/actions/chat/attachment.js

```diff
@@ -59,6 +77,10 @@ function * onLoadAttachment ({payload: {conversationIDKey, messageID, loadPrevie
   // If we are loading the actual attachment,
   // let's see if we've already downloaded it as an hdPreview
   if (!loadPreview && !isHdPreview) {
+    if (existingMessageState === 'downloading') {
```

Comment:
This seemed like the most central place to check. It's replaced by a different approach in #6726.